frontend/components/PhoneinputTest.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { Component, useRef } from "react";
```

Comment:
rename filename to `PhoneInput`? why 'Test' ?

Reply:
This file is just for Testing. The code in it is not working for web. But it is working flawlessly for Android/iOS.